Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "vob"
 description = "Vector of Bits with Vec-like API and usize backing storage"
 repository = "https://github.com/softdevteam/vob/"
-version = "0.1.0"
+version = "1.0.0"
```

Comment:
Why not 0.2.0? Is this going to be the "we consider this done" release?

Reply:
It's a backwards incompatible change, so semantic versioning says that we have to do this.

Reply:
Oh, I thought that was the second number, but you are right.

I've always considered PATCH and MINOR to be the same, but if that's how Rust does it, can't argue. Probably spent too much time with OpenBSD shared objects.

Reply:
Rust crates are morally obliged to use semantic versinoing https://semver.org/ -- I'm starting to warm up to it, even though I prefer numbers that look more normal to humans.

Reply:
Yeah, I knew the Rust guys dig semver, but I've never really looked into it. I'm not publishing crates you see ;)